pom.xml

```diff
@@ -750,7 +750,7 @@
     <profile>
       <id>scala-2.11</id>
       <properties>
-        <scala.version>2.11.8</scala.version>
+        <scala.version>2.11.12</scala.version>
```

Comment:
I think we just only need to change scala version in spark interpreter. Changing it in project root's `pom.xml` will affect all the interpreters, this may introduce other issues.

Reply:
I see all of Casandra, Flink, and Ignite will get the version upgrade because of this.
Sure, let me revert this change.